src/lib.rs

```diff
@@ -61,6 +61,9 @@ pub fn run(matches: ArgMatches) {
         header = false;
     }
 
+    let  k8s_node = get_argument(matches, "kubernetes-node");
+    println!("Running on k8s node {}", k8s_node);
```

Comment:
I added this arg so we can test the wiring. I haven't had a chance to test it yet I'll do that tomorrow.